src/nestjs-back/src/messages/messages.controller.ts

```diff
@@ -0,0 +1,65 @@
+import {
+    Body,
+    Controller,
+    Delete,
+    Get,
+    Param,
+    Patch,
+    Post
+} from '@nestjs/common';
+import { MessagesService } from './messages.service';
+import { CreateMessageDto } from './dto/create-message.dto';
+import { UpdateMessageDto } from './dto/update-message.dto';
+
+@Controller('messages')
+export class MessagesController {
+    constructor(
+        private readonly messagesService: MessagesService
+    ) {}
+
+    @Get()
+    findAll() {
+        return this.messagesService.findAll();
+    }
+
+    @Get('/channel/:channelId')
+    findAllByChannel(@Param('channelId') channelId: string) {
+        return this.messagesService.findAllByChannel(channelId);
+    }
+
+    @Get('/sender/:senderId')
```

Comment:
If you want to filter by sender use the `/messages?sender=id`.
If you want to access the sender of the message this route may make sense, but if I understand this code correctly this is only achieving filtering by sender

Reply:
See my comments on channels controller for more info